src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -95,29 +98,72 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
         }
     });
 
+    const getNoDataHeaderText = useCallback(() => {
+        if (additionalColumns.length) {
+            return 'No twins found';
```

Comment:
don't forget to loc.

Reply:
you are looking at an old version of this, should already be loc'ed